cargo-cyclonedx/Cargo.toml

```diff
@@ -24,17 +24,14 @@ lto = true
 [dependencies]
 anyhow = "1.0.62"
 cargo = "0.64.0"
-cargo_metadata = "0.15.0"
 clap = { version = "3.2.17", features = ["derive"] }
+cyclonedx-bom = { path = "../cyclonedx-bom" }
```

Comment:
We'll need a follow-up PR to set the version explicitly before we publish to crates.io. We'll also need to update the deployment process to automatically bump the version here when we publish a new release of the library (some investigation required to see how other projects handle this)